src/bosh-director/lib/bosh/director/lock.rb

```diff
@@ -129,7 +129,11 @@ def acquire
         end
 
         unless acquired
-          raise TimeoutError, "Failed to acquire lock for #{@name} uid: #{@uid}" if Time.now - started > @timeout
+          if Time.now - started > @timeout
+            lock = current_lock
+            task_id = lock ? lock.task_id : "expired"
```

Comment:
Perhaps we can say `(gone)` instead of `expired`. Parens to specifically call out there's something special with the value, and "gone" instead of expired since expired typically means a lock was lost (rather than intentionally dropped). Wondering if people will read it and come to us saying "my lock expired but the deployment still failed". Just a though; whatever you think.